profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module

```diff
@@ -264,10 +264,8 @@ function _nexteuropa_multilingual_alias_node_form_alter(&$form, &$form_state) {
       // 1. If pathauto deactivated, defining the right default value.
```

Comment:
Is this comment still valid? We don't care about pathauto being deactivated any longer in the code below.